src/system/user.cr

```diff
@@ -0,0 +1,53 @@
+require "crystal/system/user"
+
+class System::User
+  # Raised on user lookup failure.
+  class NotFoundError < Exception
+  end
+
+  extend Crystal::System::User
+
+  getter username : String
+  getter user_id : String
+  getter group_id : String
+  getter name : String
+  getter directory : String
```

Comment:
I'd prefer to call this property `home` or `home_directory`. Any thoughts on this?

Reply:
I like `home_directory`!